tests/codegen/maybeuninit-rvo.rs

```diff
@@ -0,0 +1,19 @@
+// compile-flags: -O -Cpanic=abort
```

Comment:
To make this test forward compatible, rather than using `-Cpanic=abort`, I'd suggest using `#![feature(c_unwind)]` and then have two variants, one using `extern "C"` and one using `extern "C-unwind"`. The latter would expect a memcpy, for now.

Reply:
Good idea. Let me know if I did it correctly.